### ISSUE TITLE
Eval Mode in select()

### ISSUE THREAD

**Issue by nab170130** [decile-team/distil#42]
SMI, SCMI, and SCG need to have self.model.eval() in their select() methods. Without it, the embedding computation will not necessarily be the same for similar points, which may cause performance degradation.

**Comment by nab170130** [comment]
This issue has been resolved in the latest merge to main; all strategies should now be put in eval() mode when calling select()
